React-2/Week-1/my-app/app/pages/blog_website/[id].js

```diff
@@ -0,0 +1,19 @@
+// pages/blogs/[id].js
+import { useRouter } from "next/router";
+
+export default function BlogPost() {
+  const router = useRouter();
+  const { id } = router.query;
+
+  // Convert the URL parameter to title case
+  const title = id
+    ? id.replace(/-/g, " ").replace(/\b\w/g, (char) => char.toUpperCase())
+    : "";
+
+  return (
+    <div>
+      <h1>{title}</h1>
+      <p>This is the blog post: {title}</p>
+    </div>
+  );
+}
```

Comment:
Code of the component is (almost) fine, but the page doesn't work in the end.

<img width="820" alt="image" src="https://github.com/user-attachments/assets/9da37115-3200-4a02-b536-72871783da37">

Why?

Any file in `/app` folder (except _special_ ones like `page.js` or `layout.js`) are considered just components, and will be never rendered if not included to some page, and will not affect path in any way.

The only way to create a path is to create a **folder**. And inside the folder -- a `page.js`. So it suppose to be folder `[id]` and this file suppose to be `page.js` and be inside folder `[id]`

Reply:
![image](https://github.com/user-attachments/assets/5ece8379-a73e-43dd-b38a-a72f436d2f47)